packages/ui/react-ui-mosaic/src/mosaic/Tile.tsx

```diff
@@ -43,6 +43,7 @@ export type MosaicTileProps<TData extends MosaicDataItem = MosaicDataItem, TPosi
     // TODO(burdon): Generalize events via onAction?
     onSelect?: () => void;
     onRemove?: () => void;
+    onNavigate?: () => void;
```

Comment:
Should I be using `onAction` instead? I couldn't find many other examples.

Reply:
That is a to-do that we can resolve another time, IMO.